patchdeps.py

```diff
@@ -471,12 +471,12 @@ def analyze_hunk(self, depends, patch, hunk):
 
     def print_blame(self):
         print("{}:".format(self.fname))
-        next_line = None
+        next_line = 0
         for line_state in self.line_list:
             if line_state.line is None:
                 continue
 
-            if next_line and line_state.lineno != next_line:
+            if line_state.lineno != next_line > 0:
```

Comment:
This line looks very weird and incorrect to me. I suspect that it is actually correct and it uses a generalization of the `1 < a < 10` syntax but with different operators, but that does not make for readable code IMHO. One more reason to drop this commit?